src/firefly/js/visualize/reducer/PlotView.js

```diff
@@ -87,6 +87,7 @@ import {updateTransform, makeTransform} from '../PlotTransformUtils.js';
  * @prop {boolean} [rotateNorthLock]
  * @prop {boolean} [flipYLock]
  * @prop {boolean} [useSticky]
+ * @prop {boolean} [embedMainToolbar] default to false - if true then create the main toolbar within the plot
  * @prop {boolean} [highlightFeedback]
  * @prop {boolean} [useForCoverage=false]
```

Comment:
You are already using `useXXX` as switches.  Why not follow suit and call it `useMainToolBar`.